lib/watch.js

```diff
@@ -1,4 +1,5 @@
 const path = require('node:path');
+// TODO [engine:node@>=19.1] Use `fs.watch` (or `@parcel/watcher`).
```

Comment:
To be evaluated when we upgrade, but I'm not sure we will. There seem quite a few caveats for the native watcher: https://nodejs.org/docs/latest/api/fs.html#fswatchfilename-options-listener

Reply:
I've heard quite a few projects use `@parcel/watcher` instead, and most of the caveats were resolved in 19.1 (I think the Windows stuff is outdated).